exchanges/okx/okx_websocket.go

```diff
@@ -554,8 +515,14 @@ func (ok *Okx) WsHandleData(respRaw []byte) error {
 		}
 		return fmt.Errorf("%w unmarshalling %v", err, respRaw)
 	}
-	if (resp.Event != "" && (resp.Event == "login" || resp.Event == "error")) || resp.Operation != "" {
-		ok.WsResponseMultiplexer.Message <- &resp
+	if resp.Event != "" &&
+		(resp.Event == "login" ||
+			(resp.Event == "error" &&
+				slices.Contains([]string{"60022", "60023", "60024", "60026", "63999", "60032", "60011", "60009", "60005", "60021", "60031"}, resp.Code))) {
+		// find error codes and corresponding reasons: https://www.okx.com/docs-v5/en/#error-code-websocket-public
+		if !ok.Websocket.Match.IncomingWithData("login-response", respRaw) {
```

Comment:
Unrelated, but this is the kind of thing I was talking about in #1589 , this kind of generic message of `login-response` is going to get whacky if you have multiple authenticated okx websocket connections

Reply:
Yeah good point, So there is one option that might mitigate this, when there is concurrent connections being made (which is not currently happening at the moment). We can move the stream.Match type to be held by the connection itself, which I was thinking about doing anyway because you can localise your outbound subscription response matching to the connection. So you can have this generic message on each connection and it shouldn't impede anything else. But I would need to untie a bunch of websocket Match field calls in the wrappers, so I think it will need to be in a separate PR.

Reply:
I definitely wasn't asking to hold up this PR for that. It was just a good example of potential impacts of solidifying moving things off the individual connection

Reply:
#1613 Potentially addresses this.